### PR TITLE
Fix `_generate_history_field()` for `BigAutoField`

### PR DESCRIPTION
**Pull request by tobiasmcnulty**
The way the check for `BigAutoField` is ordered in `_generate_history_field()`, it erroneously generates an `IntegerField` instead of a `BigIntegerField` in the event/history model. This won't be apparent until the project exceeds the limit for an `IntegerField` when tracking model changes, and Postgres rejects the value as out of range for an integer.

```
❯ pip freeze|grep Django              
Django==4.2.13
❯ python manage.py shell         
>>> from django.db.models import AutoField, BigAutoField
>>> isinstance(AutoField(), BigAutoField)
False
>>> isinstance(BigAutoField(), AutoField)
True
```

### CODE REVIEW

File: pghistory/core.py [hunk 1]
```diff
@@ -237,10 +237,10 @@ def _generate_history_field(tracked_model, field):
     """
     field = tracked_model._meta.get_field(field)
 
-    if isinstance(field, models.AutoField):
-        return models.IntegerField()
-    elif isinstance(field, models.BigAutoField):  # pragma: no cover
+    if isinstance(field, models.BigAutoField):
```
Comment (Mariatta) [inline pghistory/core.py:240]: Looks like there was no test for it before. So either we can write the test now, or add back the `# pragma: no cover`.

File: settings.py [hunk 2]
```diff
@@ -1,8 +1,5 @@
-import os
```
Comment (tobiasmcnulty) [inline settings.py:1]: `make lint` complained about these lines.

**Review by gsimore — APPROVED** [review]
Nice changes! Looks good to me

**Review by Mariatta — APPROVED** [review]
Looks great. Thanks for adding tests.

**Comment by wesleykendall** [comment]
Wow thanks so much for this fix and the tests! I will be sure to put a disclaimer in the release notes that users may see new migrations if they were previously using BigAutoFields in tracked models

**Comment by wesleykendall** [comment]
Released in 3.3.0

**Pull request merged**